src/main/java/pl/piekoszek/gorskimatches/challange/Challenge.java

```diff
@@ -0,0 +1,83 @@
+package pl.piekoszek.gorskimatches.challange;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+public class Challenge {
+    @Id
+    private UUID uuid;
+
+    private String email;
+
+    private int registeredUserScore;
+
+    private int nonRegisteredUserScore;
+
+    private float nonRegisteredUserTime;
```

Comment:
It would be better to have unit in variable name, like `nonRegisteredUserTimeSeconds` or `nonRegisteredUserTimeMinutes`